code/__DEFINES/antagonists.dm

```diff
@@ -397,3 +397,6 @@ GLOBAL_LIST_INIT(human_invader_antagonists, list(
 #define SPY_DIFFICULTY_MEDIUM "Medium"
 /// Very difficult to accomplish, almost guaranteed to require crew conflict
 #define SPY_DIFFICULTY_HARD "Hard"
+
+/// Camera network nuclear operative bodycams are on
+#define OPERATIVE_CAMERA_NET "operative"
```

Comment:
Put this in the file with all the other camera nets please

Reply:
Do we actually have a file with all the camera nets in the same place? __DEFINES/camnets.dm doesn't actually have any cam nets and the recent bitrunning camnet was in the bitrunning defines. Should I just move those two to camnets.dm?

Reply:
I added it in the PR that removed camera assembly structures

Reply:
Ahh, couldn't find it because my branch wasn't up to date.